app/assets/stylesheets/course/achievement.scss

```diff
@@ -1,3 +1,12 @@
 // Place all the styles related to the course/achievement controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.course-achievements {
+  .achievement-list, .new_achievement {
```

Comment:
Selector `new_achievement` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line